spikes/kafka-flink/README.Windows.rst

```diff
@@ -0,0 +1,104 @@
+#################################################
+Apache Kafka, Apache Flink and CrateDB on Windows
+#################################################
+
+*****
+About
+*****
+
+This document will outline how to run the tutorial on Windows 10, using PowerShell.
+On Windows 10, please use those commands with PowerShell 3+ (pwsh) and `Docker Desktop on Windows`_.
+
+
+*****
+Setup
+*****
+
+Prepare a sandbox directory::
+
+    New-Item sandbox/kafka-flink-cratedb -ItemType Directory -ErrorAction SilentlyContinue
+    cd ./sandbox/kafka-flink-cratedb/
+
+
+Infrastructure
+==============
+
+In order to start Kafka, Flink and CrateDB, invoke::
+
+    Set-Variable -Name DOCKER_COMPOSE_URL -value "https://raw.githubusercontent.com/crate/cratedb-examples/amo/kafka-flink/spikes/kafka-flink/docker-compose.yml"
+    Invoke-WebRequest -Uri "${DOCKER_COMPOSE_URL}" -OutFile "docker-compose.yml"
+    docker-compose up
```

Comment:
similar to comment above,
in case of docker-compose failure, it might be an issue with small vm.max_map_count - running command
sysctl -w vm.max_map_count=262144. It has to be run from WSL, so in case of Powerschell it's 
`wsl`
`sysctl -w vm.max_map_count=262144`

Reply:
So, while the recipe for Windows is running in "native" mode, the `sysctl` command has yet to be invoked within WSL?

Reply:
Thank you. I just added d2a8043c0d9, can you have a look?

Reply:
> So, while the recipe for Windows is running in "native" mode, the `sysctl` command has yet to be invoked within WSL?

yes, that's how it worked on my machine - I left a comment about that below